package.json

```diff
@@ -150,15 +150,15 @@
     "@nestjs/cli": "^9.4.0",
     "@nestjs/schematics": "^10.1.1",
     "@nestjs/testing": "^10.3.2",
-    "@nx/cypress": "^17.1.3",
-    "@nx/eslint-plugin": "^16.9.1",
-    "@nx/jest": "^16.9.1",
-    "@nx/js": "^16.9.1",
-    "@nx/next": "^16.9.1",
-    "@nx/node": "^18.0.2",
-    "@nx/react": "^16.9.1",
-    "@nx/storybook": "^17.1.3",
-    "@nx/workspace": "^16.9.1",
+    "@nx/cypress": "18.0.3",
+    "@nx/eslint-plugin": "18.0.3",
+    "@nx/jest": "18.0.3",
+    "@nx/js": "18.0.3",
+    "@nx/next": "18.0.3",
+    "@nx/node": "18.0.3",
+    "@nx/react": "18.0.3",
+    "@nx/storybook": "18.0.3",
+    "@nx/workspace": "18.0.3",
```

Comment:
Did we mean to remove the semver range operators on all these?
Not that it's wrong or anything if we want to keep them all in sync and update them manually, just confirming.

Reply:
The nx cli `migrate` command seems to do it like this. Ideally we'd like to keep using the nx cli to update everything in one go.